lib/super_good/solidus_taxjar/backfill_transactions.rb

```diff
@@ -2,6 +2,13 @@ module SuperGood
   module SolidusTaxjar
     class BackfillTransactions
       def self.call
+        backfilled_orders = []
+        ::Spree::Order.complete.find_each do |order|
+          next if order.taxjar_order_transactions.any?
```

Comment:
I haven't looked at what this association is but could we get the database to do this instead of looping over them all and skipping the ones that have transactions?

Reply:
I could imagine a more efficient query where we only grab orders where the `COUNT` of TaxJar order transactions is `0`, but this is a "run it once" kind of process, so probably not critical to optimize it.

Reply:
It might be worth noting that orders are completed before they are shipped. So not all complete orders are liable for tax.

Reply:
@benjaminwil good catch! Updated to also check for an orders ship state.